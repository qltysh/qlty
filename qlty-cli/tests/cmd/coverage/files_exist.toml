args = [
  "coverage",
  "publish",
  "--dry-run",
  "--override-commit-sha",
  "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67",
  "--override-build-id",
  "123",
  "lcov.info",
  "--print",
  "--json",
  "--skip-missing-files",
  "--summary"
]
bin.name = "qlty"

[env]
remove = ["GITHUB_ACTIONS"]

[env.add]
QLTY_COVERAGE_TOKEN = "123"
