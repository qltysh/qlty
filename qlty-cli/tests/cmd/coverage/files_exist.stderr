qlty [..] [..]-[..] ([..] debug [..])
https://qlty.sh/d/coverage

  Retrieving CI metadata...
  → unknown CI commit "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67" on branch ""

  Reading code coverage data...
  → Found 1 files with code coverage data

  Exporting code coverage data...
  → Exported to "tmp/qlty-coverage"
